pySearch.py

```diff
@@ -3,33 +3,71 @@
 import urllib
 import argparse
 import webbrowser
+import re
 
 argparser = argparse.ArgumentParser()
-argparser.add_argument("-s", help="Takes a query to search for and searches it.", nargs="*")
+argparser.add_argument("-s", action="append", help="Takes a query to search for and searches it.", nargs="*")
 argparser.add_argument("-e", "--engine", help="Changes the name or alias of a search engine and sets it as the search engine for the session", nargs="+")
 argparser.add_argument("-d", "--domain", help="Changes the domain extention", nargs="+")
+argparser.add_argument("-b", "--browser", help="Changes the web browser", nargs="+")
 
 args = argparser.parse_args()
 
+
+    
+#print available broswers
+def printAvailableBrowsers(invalid):
+        print("You have selected an invalid or unreigstered browser: " + invalid + ".\nHere is a list of available browsers")
+        for i in webbrowser._browsers:
```

Comment:
Code nit, let's avoid using a generic iterator like `i`. Iterators in Python's for loops act as the actual object so I think we should use a variable that's more informative to maintain the readability of the code. `i` in particular is an incredibly shaky choice when `i` is the generic `int` iterator variable in other languages. I'm worried that someone new to Python may look at the code and assume `i` is an int.

Instead `for browser in webbrowser._browsers:` gives a much better indication of what's going on.